### DISCUSSION TITLE
oliveshell运行命令式如何调用变量$FILE_PATH ？

### DISCUSSION THREAD

**Discussion by yburger** [go-olive/olive#47] [Q&A] [answered]
我需要在录制结束后使用oliveshell运行`rclone copy $FILE_PATH anime:LiveRecords`的命令，希望能在这条命令中能够解析出$FILE_PATH变量的值并替换到命令中，我尝试了这样写
```
[[Shows.PostCmds]]
Path = "oliveshell"
Args = ["rclone", "copy", "$FILE_PATH", "anime:LiveRecords"]
```
但是最后运行的命令中$FILE_PATH没有解析出来路径，而是直接带着变量名运行了，那么如何写配置文件才能解析出变量值呢？

**Comment by luxcgo** [comment]
@UVJkiNTQ 这位大佬应该深有体会。。

**Comment by luxcgo** [comment, marked as answer]
环境变量只有在类sh下才能正确解析，且zsh相比普通sh对文件名中有空格的情况支持会更好。
故推荐写法如下
```sh
[[Shows.PostCmds]]
Path = "oliveshell"
Args = ["/bin/zsh", "-c", "rclone copy $FILE_PATH anime:LiveRecords"]
```

**Comment by yburger** [comment]
> 环境变量只有在类sh下才能正确解析，且zsh相比普通sh对文件名中有空格的情况支持会更好。 故推荐写法如下
> 
> ```shell
> [[Shows.PostCmds]]
> Path = "oliveshell"
> Args = ["/bin/zsh", "-c", "rclone copy $FILE_PATH anime:LiveRecords"]
> ```
谢谢解答，这样确实可以了
顺便问一句，有计划添加同时录制弹幕的功能吗？
